src/index.js

```diff
@@ -34,6 +36,18 @@ export class QueryList {
     toObject() {
         return sortBy(this.queries, query => query.constraints !== null);
     }
+
+    static sort(queries) {
+        const sortedQueries = [];
+
+        Object.entries(queries)
+            .sort((a, b) => QueryUtils.sort(a[1].constraints, b[1].constraints))
+            .forEach((query) => {
```

Comment:
If queryList is empty this will not work

Reply:
queryList will default to an empty object and Object.entries allows empty objects sooooo..... yeah, not so much of a problem ;)